packages/spacecat-shared-data-access/src/models/fix-entity/fix-entity.schema.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2025 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { isNonEmptyObject } from '@adobe/spacecat-shared-utils';
+
+import SchemaBuilder from '../base/schema.builder.js';
+import FixEntity from './fix-entity.model.js';
+import FixEntityCollection from './fix-entity.collection.js';
+import { Suggestion } from '../suggestion/index.js';
+
+/*
+Schema Doc: https://electrodb.dev/en/modeling/schema/
+Attribute Doc: https://electrodb.dev/en/modeling/attributes/
+Indexes Doc: https://electrodb.dev/en/modeling/indexes/
+ */
+
+const schema = new SchemaBuilder(FixEntity, FixEntityCollection)
+  .addReference('belongs_to', 'Suggestion', ['status'])
+  .addAttribute('type', {
+    type: Object.values(Suggestion.TYPES),
+    required: true,
+    readOnly: true,
+  })
+  .addAttribute('executedBy', {
```

Comment:
`executedAt` would also be nice. Also `createdAt`, `publishedAt`, `rolledBackAt`, or generally keeping track of status changes ;)

Reply:
@phornig `createdAt`and `updatedAt` are tracked by the `BaseModel`